src/main/java/com/hulu/ftl/formats/XMLFormat.java

```diff
@@ -38,7 +38,7 @@ public XMLFormat(InputStream stream) throws IOException {
         String[] rootSelectors = selectors.clone();
 
         for(int x=0; x<rootSelectors.length; x++) {
-            rootSelectors[x] = "/*/" + rootSelectors[x];
+            rootSelectors[x] = rootSelectors[x].equals("/") ? "/" : "/*/" + rootSelectors[x];
```

Comment:
This is so people can reference the root element (in this case program) just as "/"